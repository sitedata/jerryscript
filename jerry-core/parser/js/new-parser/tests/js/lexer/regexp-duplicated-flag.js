/* This test contains regexp literal parser tests. */

	/ab/iggm
