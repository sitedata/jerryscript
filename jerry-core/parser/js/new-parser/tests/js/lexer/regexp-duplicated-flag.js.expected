
--- Script parsing start ---


--- Script parsing end ---

Parse error 'Duplicated regexp flag.' at line: 3 col: 9
